lib/gooddata/lcm/actions/synchronize_users.rb

```diff
@@ -49,6 +49,7 @@ def call(params)
           domain_name = params.organization || params.domain
           project = client.projects(params.gdc_project) || client.projects(params.gdc_project_id)
           data_source = GoodData::Helpers::DataSource.new(params.input_source)
+          data_product = params.data_product
```

Comment:
Refers to 'params' more than self (maybe move it to another class?) (FeatureEnvy)